process/process_areas/quality_management/quality_roles.rst

```diff
@@ -0,0 +1,59 @@
+..
+   # *******************************************************************************
+   # Copyright (c) 2025 Contributors to the Eclipse Foundation
+   #
+   # See the NOTICE file(s) distributed with this work for additional
+   # information regarding copyright ownership.
+   #
+   # This program and the accompanying materials are made available under the
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+Roles
```

Comment:
Out of curiosity, why do we extract the quality manager role from the normal role description? (Just wanted to know, as I may have missed the discussion)